addon/utils/e3-easing.js

```diff
@@ -0,0 +1,16 @@
+const EASING = {
+  'ease-in-cubic': function(percent) {
```

Comment:
Could this just be named `cubic` instead of `ease-in-cubic`, or does that have a special meaning?  Seems out of place compared to the other two.